src/providerhub/HISTORY.rst

```diff
@@ -3,6 +3,24 @@
 Release History
 ===============
 
+0.2.0
+++++++
+* Support commands for
+`az providerhub notification-registration`
+`az providerhub operation`
+`az providerhub sku`
+
+* Enable creation of nested resource types and adding extension categories.
+
+Refine and update initial release commands.
+* [BREAKING CHANGE] `az providerhub manifest checkin`:
+`--arm-manifest-location` renamed to `--baseline-arm-manifest-location` or `--manifest-location`
+* [BREAKING CHANGE] `az providerhub default-rollout create`:
+`--row2-wait-duration` moved to `--row2 wait-duration`
+`--skip-regions` moved to `--canary skip-regions`
```

Comment:
You can flatten --row2 and --canary parameters and expose sub properties as top level argument. https://github.com/Azure/autorest.az#flatten-a-parameter

Reply:
Thanks for the suggestions, added support for these parameters to avoid breaking change